### PR TITLE
docs: add note on AggregateError to README of browser chapter 02

### LINKED ISSUE #25: What address to use for multiaddress in the chapter 02?
Hi! I am a beginner in lib2p and peer-to-peer tech in general. I'm trying the chapter 02.
I tried several addresses including the one from the chapter 03.
```
/ip4/127.0.0.1/tcp/63786/ws/ipfs/QmWjz6xb8v9K4KnYEwP5Yk75k5mMBCehzWFLCvvQpYxF3d
/ip4/0.0.0.0/tcp/15555/ws/p2p-webrtc-star/p2p/QmWjz6xb8v9K4KnYEwP5Yk75k5mMBCehzWFLCvvQpYxF3d
/ip4/192.168.1.205/tcp/63786/ws
```
The error for the first 2 addresses is:
```
Download the React DevTools for a better development experience: https://fb.me/react-devtools react-dom.development.js:24994:16
Getting our PeerId App.js:26:14
Creating our Libp2p instance App.js:34:16
AggregateError: 
    parcelRequire<["../node_modules/multistream-select/src/select.js"]</module.exports@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:118789:17
    async*select@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:118909:12
    _encryptOutbound@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:119685:21
    upgradeOutbound@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:119447:22
    dial@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:133719:39
    async*dial@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:102821:30
    dialAction@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:106492:36
    run/<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:106296:29
    Async*run@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:106289:36
    _createPendingDial@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:106508:28
    connectToPeer@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:106414:71
    _maybeConnect@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:122213:29
    _onDidStart/<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:122166:12
    emit@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:29160:17
    add@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:120027:16
    _createDialTarget@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:106455:34
    connectToPeer@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:106408:29
    dialProtocol@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:122026:38
    dial@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:122003:17
    createLibp2p@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:145687:18
    async*App/</<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:99659:28
    App/<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:99662:9
    commitHookEffectListMount@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:23768:30
    commitPassiveHookEffects@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:23805:40
    callCallback@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:4350:18
    invokeGuardedCallbackDev@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:4399:20
    invokeGuardedCallback@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:4452:35
    flushPassiveEffectsImpl@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:26869:34
    unstable_runWithPriority@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:3598:16
    runWithPriority$1@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:15237:14
    flushPassiveEffects@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:26837:16
    commitBeforeMutationEffects/<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:26716:15
    workLoop@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:3542:46
    flushWork@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:3498:20
    performWorkUntilDeadline@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:3102:52
    EventHandlerNonNull*parcelRequire<["../node_modules/scheduler/cjs/scheduler.development.js"]</<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:3126:7
    parcelRequire<["../node_modules/scheduler/cjs/scheduler.development.js"]<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:3795:5
    newRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:47:24
    localRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:53:14
    parcelRequire<["../node_modules/scheduler/index.js"]<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:3803:20
    newRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:47:24
    localRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:53:14
    parcelRequire<["../node_modules/react-dom/cjs/react-dom.development.js"]</<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:4180:28
    parcelRequire<["../node_modules/react-dom/cjs/react-dom.development.js"]<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:28979:5
    newRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:47:24
    localRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:53:14
    parcelRequire<["../node_modules/react-dom/index.js"]<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:29017:20
    newRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:47:24
    localRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:53:14
    parcelRequire<["index.js"]<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:145705:40
    newRequire@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:47:24
    parcelRequire<@http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:81:17
    @http://localhost:1234/03-Muxing-Encryption.e31bb0bc.js:120:3
    AggregateError index.js:33
    maybeSettle index.js:31
    pSome index.js:69
    pSome index.js:51
    _promise index.js:61
    PCancelable index.js:31
    pSome index.js:7
    exports index.js:6
    run dial-request.js:53
    _createPendingDial index.js:167
    connectToPeer index.js:76
    _maybeConnect index.js:509
    _onDidStart index.js:466
    emit events.js:152
    add address-book.js:134
    _createDialTarget index.js:112
    connectToPeer index.js:71
    dialProtocol index.js:324
    dial index.js:305
    createLibp2p libp2p.js:34
    App App.js:35
    App App.js:33
    React 6
    unstable_runWithPriority scheduler.development.js:653
    React 3
    workLoop scheduler.development.js:597
    flushWork scheduler.development.js:552
    performWorkUntilDeadline scheduler.development.js:164
    EventHandlerNonNull*parcelRequire<["../node_modules/scheduler/cjs/scheduler.development.js"]</< scheduler.development.js:187
    parcelRequire<["../node_modules/scheduler/cjs/scheduler.development.js"]< scheduler.development.js:15
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    parcelRequire<["../node_modules/scheduler/index.js"]< index.js:6
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    React 2
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    parcelRequire<["../node_modules/react-dom/index.js"]< React
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    parcelRequire<["index.js"]< index.js:3
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    parcelRequire 03-Muxing-Encryption.e31bb0bc.js:81
    <anonymous> 03-Muxing-Encryption.e31bb0bc.js:120
libp2p.js:37:12
    createLibp2p libp2p.js:37
    App App.js:35
    App App.js:33
    React 6
    unstable_runWithPriority scheduler.development.js:653
    React 3
    workLoop scheduler.development.js:597
    flushWork scheduler.development.js:552
    performWorkUntilDeadline scheduler.development.js:164
    (Async: EventHandlerNonNull)
    parcelRequire<["../node_modules/scheduler/cjs/scheduler.development.js"]</< scheduler.development.js:187
    parcelRequire<["../node_modules/scheduler/cjs/scheduler.development.js"]< scheduler.development.js:15
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    parcelRequire<["../node_modules/scheduler/index.js"]< index.js:6
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    React 2
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    parcelRequire<["../node_modules/react-dom/index.js"]< React
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    localRequire 03-Muxing-Encryption.e31bb0bc.js:53
    parcelRequire<["index.js"]< index.js:3
    newRequire 03-Muxing-Encryption.e31bb0bc.js:47
    parcelRequire 03-Muxing-Encryption.e31bb0bc.js:81
    <anonymous> 03-Muxing-Encryption.e31bb0bc.js:120
```

Bootstrap starting log:
```
$ node index.js 
Signaling server running at /ip4/0.0.0.0/tcp/15555/ws/p2p-webrtc-star
Node started with addresses:
/ip4/0.0.0.0/tcp/15555/ws/p2p-webrtc-star/p2p/QmWjz6xb8v9K4KnYEwP5Yk75k5mMBCehzWFLCvvQpYxF3d
/ip4/127.0.0.1/tcp/63785
/ip4/192.168.1.205/tcp/63785
/ip4/127.0.0.1/tcp/63786/ws
/ip4/192.168.1.205/tcp/63786/ws

Node supports protocols:
/libp2p/circuit/relay/0.1.0
/ipfs/id/1.0.0
/ipfs/id/push/1.0.0
/ipfs/ping/1.0.0
/libp2p/chat/1.0.0
/meshsub/1.0.0
/floodsub/1.0.0
/ipfs/kad/1.0.0
```

### PR DESCRIPTION
**Pull request by vogdb**
To resolve #25

### CODE REVIEW

File: chat/browser/03-Muxing-Encryption/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Chapter 03 - Muxing and Encryption
 
+**Note**: Expect to have an AggregateError in console as your connection needs to be upgraded. See [#25](/../../issues/25) for more details.
+
```
Comment (jacobheun) [inline chat/browser/03-Muxing-Encryption/README.md:4]: For Chapter 03, you shouldn't get an error once it's completed, it should connect just fine. Were you hitting errors here?

File: chat/browser/02-Multiaddrs/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Chapter 02 - Multiaddrs
 
-**Note**: For this chapter, you can get the multiaddr of the peer to dial by running the Bootstrap node as mentioned in the [browser README](../README.md). Running the node will result in the Bootstrap addresses being printed.
+**Note**: For this chapter, you can get the multiaddr of the peer to dial by running the Bootstrap node as mentioned in the [browser README](../README.md). Running the node will result in the Bootstrap addresses being printed. Also expect to have an AggregateError in console as your connection needs to be upgraded. See [#25](/../../issues/25) for more details.
```
Comment (jacobheun) [inline chat/browser/02-Multiaddrs/README.md:3]:
```suggestion
**Note**: For this chapter, you can get the multiaddr of the peer to dial by running the Bootstrap node as mentioned in the [browser README](../README.md). Running the node will result in the Bootstrap addresses being printed. Also expect to have an AggregateError in console as you will need to complete Chapter 03 before a full connection can be established. See [#25](/../../issues/25) for more details.
```
```suggestion
**Note**: For this chapter, you can get the multiaddr of the peer to dial by running the Bootstrap node as mentioned in the [browser README](../README.md). Running the node will result in the Bootstrap addresses being printed. Also expect to have an AggregateError in console as your connection needs to be upgraded. See [#25](/../../issues/25) for more details.
```

File: chat/browser/03-Muxing-Encryption/README.md (hunk 1 above)
Comment (vogdb) [reply chat/browser/03-Muxing-Encryption/README.md]: My bad. Fixed it.

**Comment by vogdb** [comment]
Now I've noticed there is `browser/help` folder with some useful help. It is not mentioned anywhere. Should I mention about it in `browser/README.md`?
Also I hope we can squash before merging?

**Comment by jacobheun** [comment]
> Now I've noticed there is browser/help folder with some useful help. It is not mentioned anywhere. Should I mention about it in browser/README.md?

Yes that would be great if you want to include that here.

> Also I hope we can squash before merging?

I'll make sure to squash

**Review by jacobheun — APPROVED** [review]
LGTM, thanks again for the PR! ❤️

**Comment by vogdb** [comment]
Thank you for guidance

**Pull request merged**